grpc/src/compiler/go_generator.cc

```diff
@@ -0,0 +1,435 @@
+/*
```

Comment:
Yes, it has been completely compatible with grpc-go. The compiler is not maintained by grpc-go though, it's taken care by the proto-go -  https://github.com/golang/protobuf/tree/master/protoc-gen-go